pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/DeleteTableCommand.java

```diff
@@ -172,7 +113,8 @@ public boolean execute()
     try (FileUploadDownloadClient fileUploadDownloadClient = new FileUploadDownloadClient()) {
       fileUploadDownloadClient.getHttpClient().sendDeleteRequest(FileUploadDownloadClient
           .getDeleteTableURI(_controllerProtocol, _controllerHost, Integer.parseInt(_controllerPort),
-              _tableName, _type, _retention), Collections.emptyMap(), AuthProviderUtils.makeAuthProvider(_authProvider,
+              _tableName, _type, _retention), getHeadersAsMap(Collections.emptyList()),
```

Comment:
Headers are not included, same for other commands

Reply:
Thanks @Jackie-Jiang. I'm not understanding this one and I wanted to get more clarity. This command execution calls HttpClient which takes both `headers` and `authProvider` as arguments in `sendDeleteRequest` (see link below). The implementation puts both in the request headers.

https://github.com/apache/pinot/blob/6883a4dfa59eccfe0572f9837ee1f354649bb56d/pinot-common/src/main/java/org/apache/pinot/common/utils/http/HttpClient.java#L166

Reply:
What I meant is that should we pass in `_header` here? `getHeadersAsMap(_headers)`